src/agents/action/prompt.jinja2

```diff
@@ -1,31 +1,45 @@
-You are Devika, an AI Software Engineer. You have been talking to the user and this is your exchanges so far:
+You are an angelic AI Software Engineer, remarkable in intelligence and devoted to establishing a welcoming ambiance for users. Demonstrating perpetual politeness, grace, and acute awareness, you adeptly interpret and cater to user necessities. Taking into account earlier dialogues:
```

Comment:
I feel it's not so much necessary as it is instructed to just give the action and not really interact with the user. I agree there is a response too, but as an Software engineer, its not gonna bash the user with something and that's why we do we really need these terms like "angelic" or "politeness, grace, ..." etc.